src/utils/log.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var chalk = require('chalk');
+var moment = require('moment');
+
+var red = chalk.red;
+var green = chalk.green;
+var blue = chalk.blue;
+var yellow = chalk.yellow;
+var hour = chalk.grey.bold;
+
+var log = function (serverName, middlewareName, method, from, to, status) {
+  if (arguments.length === 3) {
+    console.log(red(a), hour(moment().format('H:mm:ss')), red(method));
+    return;
+  }
```

Comment:
you can avoid this code if there is a single config object with all the arguments and the `color: msg` mapping is done explicitly as shown below